packages/typescriptlang-org/src/templates/play.tsx

```diff
@@ -51,85 +54,141 @@ const Play: React.FC<Props> = (props) => {
-        // Importing "vs/language/typescript/tsWorker" will set ts as a global
-        const ts = (global as any).ts
-        const isOK = main && ts && sandbox && playground
-        if (isOK) {
-          document.getElementById("loader")!.parentNode?.removeChild(document.getElementById("loader")!)
-        } else {
-          console.error("Errr")
-          console.error("main", !!main, "ts", !!ts, "sandbox", !!sandbox, "playground", !!playground)
-        }
-
-        // Set the height of monaco to be either your window height or 600px - whichever is smallest
-        const container = document.getElementById("playground-container")!
-        container.style.display = "flex"
-        const height = Math.max(window.innerHeight, 600)
-        container.style.height = `${height - Math.round(container.getClientRects()[0].top) - 18}px`
-
-        // Create the sandbox
-        const sandboxEnv = await sandbox.createTypeScriptSandbox({
-          text: i("play_default_code_sample"),
-          compilerOptions: {},
-          domID: "monaco-editor-embed",
-          useJavaScript: !!params.get("useJavaScript"),
-          acquireTypes: !localStorage.getItem("disable-ata"),
-          logger: {
-            error: console.error,
-            log: console.log
-          },
-        }, main, ts)
-
-        const playgroundConfig = {
-          lang: props.pageContext.lang,
-          prefix: withPrefix("/")
-        }
-
-        playground.setupPlayground(sandboxEnv, main, playgroundConfig, i as any, React)
-
-        // Dark mode faff
-        const darkModeEnabled = window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)')
-        if (darkModeEnabled.matches) {
-          sandboxEnv.monaco.editor.setTheme("sandbox-dark");
+      })
+
+      re(
+        [
+          "vs/editor/editor.main",
+          "vs/language/typescript/tsWorker",
+          "typescript-sandbox/index",
+          "typescript-playground/index",
+        ],
+        async (
+          main: typeof import("monaco-editor"),
+          tsWorker: any,
+          sandbox: typeof import("typescript-sandbox"),
+          playground: typeof import("typescript-playground")
+        ) => {
+          // Importing "vs/language/typescript/tsWorker" will set ts as a global
+          const ts = (global as any).ts
+          const isOK = main && ts && sandbox && playground
+          if (isOK) {
+            document
+              .getElementById("loader")!
+              .parentNode?.removeChild(document.getElementById("loader")!)
+          } else {
+            console.error("Errr")
+            console.error(
+              "main",
+              !!main,
+              "ts",
+              !!ts,
+              "sandbox",
+              !!sandbox,
+              "playground",
+              !!playground
+            )
+          }
```

Comment:
ergh - from 9 readable lines of code to 30 

I'll add this file to the prettier ignore, and merge your changes manually if you accept the CLA

Reply:
Oops... didn't think to check if prettier formatted it. If it's easier to do anything on my end, just let me know and I'll be happy to.

Reply:
Nah, I'll merge your changes in manually now and deploy straight away - I already have a few changes sitting by